.about.yml

```diff
@@ -0,0 +1,109 @@
+---
+# .about.yml project metadata
+#
+# Short name that acts as the project identifier (required)
+name: C2
+
+# Full proper name of the project (required)
+full_name: C2
+
+# The type of content in the repo
+# values: app, docs, policy
+type: app
+
+# Describes whether a project team, working group/guild, etc. owns the repo (required)
+# values: guild, working-group, project
+owner_type: project
+
+# Name of the main project repo if this is a sub-repo; name of the working group/guild repo if this is a working group/guild subproject
+parent: C2
+
+# Maturity stage of the project (required)
+# values: discovery, alpha, beta, live
+stage: live
+
+# Whether or not the project is actively maintained (required)
+# values: active, deprecated
+status: active
+
+# Description of the project
+description: an approval process automation tool
+
+# Should be 'true' if the project has a continuous build (required)
+# values: true, false
+testable: true
+
+# Team members contributing to the project (required)
+# Items:
+# - github: GitHub user name
+#   id: Internal team identifier/user name
+#   role: Team member's role; leads should be designated as 'lead'
+team:
+- github: afeld
```

Comment:
Technically you don't need _both_ `github:` and `id:` here, but so long as it's certain that they're both correct, there's no harm done.

I'd also maybe push the `lead:` to the top, but it doesn't really matter, if you want to keep things alphabetized as you have them here.